Migration.md

```diff
@@ -5,6 +5,20 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo Rendering 9.x to 10.x
+
+### Removals
+
+1. The environment variable `GZ_RENDERING_RESOURCE_PATH` is removed. Use
+`gz::rendering::getResourcePath()` instead.
+
+1. The environment variable `GZ_RENDERING_ENGINE_INSTALL_DIR` is removed. Use
+`gz::rendering::getEngineInstallDir()` instead.
```

Comment:
We're still reading the env vars so just clarifying that it's only the macros that are removed:

```suggestion
1. The macro `GZ_RENDERING_RESOURCE_PATH` is removed. Use
`gz::rendering::getResourcePath()` instead.

1. The macro `GZ_RENDERING_ENGINE_INSTALL_DIR` is removed. Use
`gz::rendering::getEngineInstallDir()` instead.
```